osler/workup/tests/test_scripts.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase
+from osler.workup.models import Workup
+from osler.workup.scripts import workup_date_checker
+from .factories import WorkupFactory
+import datetime
+
+class testScripts(TestCase):
+
+    fixtures = ['workup', 'core']
+
+    def test_workup_date_checker(self):
```

Comment:
I don't understand how this test works when you don't specify the encounter date for the test workups.

Reply:
You should add an additional tests to check individual functions, such as whether the differ by n days function works for exactly n days